pkgs/applications/version-management/codeberg-cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, CoreServices
+, Security
+, fetchFromGitea
+, openssl
+, pkg-config
+, rustPlatform
+, stdenv
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "codeberg-cli";
+  version = "0.3.5";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "RobWalt";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-KjH78yqfZoN24TBYyFZuxf7z9poRov0uFYQ8+eq9p/o=";
+  };
+
+  cargoSha256 = "sha256-RE4Zwa5vUWPc42w5GaaYkS6fLIbges1fAsOUuwqR2ag=";
```

Comment:
Can we use `cargoHash` instead of `cargoSha256`?

Reply:
Most certainly!

Reply:
What's the difference if I may ask? Sorry I'm still a nix newbie